app/src/main/java/com/github/mik629/android/fundamentals/MovieDetailsActivity.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.mik629.android.fundamentals
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.github.mik629.android.fundamentals.databinding.ActivityMovieDetailsBinding
+
+class MovieDetailsActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMovieDetailsBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMovieDetailsBinding.inflate(layoutInflater)
+
+        setContentView(binding.root)
+        binding.back.setOnClickListener {
+            finish()
```

Comment:
I would've used `cicerone` navigation library and `router.exit()` to go back to previous screen